src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -20,6 +20,11 @@ sirius.isEmpty = function (value) {
     }
 
     if (sirius.isDefined(value) && typeof value === 'object') {
+        // Check if it's a DOM object (has a nodeType property)
+        if (value.nodeType !== undefined) {
+            return false;
```

Comment:
A meaningful empty check for DOM elements could be implemented here. Otherwise I would actually prefer to throw an error as always returning `false` isn't correct either.

A meaningful logic could be to check whether the list of child Nodes and the textContent are empty